### ISSUE TITLE
'Open ports on router’ option dissapears when there is not firewall zone called 'wan'

### ISSUE THREAD

**Issue by shoulders** [openwrt/luci#2056]
Hi

FIrst of all I have been configuring my BT Home Hub v5A router to be a wireless client that gives a NAT network on the 4 yellow ethernets and my own private network through the red socket. This all works fine but the reason I tell you this is that I renamed all of the zones so they were relevant to my setup hence why I renamed the default **wan** to **nat_wwan**

**Open ports on router** missing  (Network --> Firewall --> Traffic Rules)

when I rename the **wan** zone to **nat_wwan** the option 'Open ports on router’ option dissapears. I can get it to reappear by either renaming this zone back or creating a new zone with the name **wan**.

Also rules I have added do work even if this option is not here. (i.e. I added them before renaming the zone)

How it should look
![image](https://user-images.githubusercontent.com/319997/43993647-ba6f62c2-9d88-11e8-9e7d-a3534fc5e03a.png)

What it looks like without the **wan** zone
![image](https://user-images.githubusercontent.com/319997/43993654-e6ef0884-9d88-11e8-95c8-7547891bfa03.png)

So basically there is some sort of dependance on having a zone named **wan**, can this be fixed/removed.

My Setup
- Firmare version  = OpenWrt 18.06.0 r7188-b0b5c64c22 / LuCI openwrt-18.06 branch (git-18.210.69179-6df9a57) - (18.06 Stable)
- Router = BT Home Hub 5A (it is actually a Plusnet one)
- if my configuration is of note I can supply that aswell

Thanks

**Comment by dibdot** [comment]
This should work most probably with a named zone section, e.g. (untested!):

```
config zone 'wan'
        option name 'nat_wwan'
        option input 'REJECT'
        option output 'ACCEPT'
        option forward 'REJECT'
        option masq '1'
        option mtu_fix '1'
        option network 'trm_wwan'
```

**Comment by shoulders** [comment]
@dibdot 

This is a workaround that you have to use the SSH to fix rather than using than using LuCI.

thanks anyway

**Comment by shoulders** [comment]
@jow- thanks for the quick update

for reference I am running my OpenWRT as a wireless client (x2) which I named **nat_wwan** and **bridge_wwan** which are both wan connections so to keep things neat I used appropriate names and deleted the **wan** interface.
